code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -278,7 +278,7 @@
 	energy = 100
 	max_energy = 100
 	desc = "A technological marvel, supposedly able to mix just the mixture you'd like to drink the moment you ask for one."
-	dispensable_reagents = list("ice","cream", "cider", "beer","kahlua","whiskey","wine","vodka","gin","rum","tequilla","vermouth","cognac","ale","mead")
+	dispensable_reagents = list("ice","cream", "cider", "beer","kahlua","whiskey","wine","vodka","gin","rum","tequilla","vermouth","cognac","ale","mead","synthanol")

```

Comment:
Oh god the spacing, not only is it absent, but it's inconsistent!

Dare I...

:eyes: 
I DARE